app/routes/app-detected.js

```diff
@@ -19,6 +19,7 @@ export default Route.extend({
       this.applicationBooted = ({ booted }) => {
         if (booted) {
           port.off('general:applicationBooted', this.applicationBooted);
+          this.applicationBooted = null;
```

Comment:
In `deactivate` it checks for this variable to avoid calling `off` twice, but we never unset it.